docs/tutorials/influxdb-alerting/add-alerting.md

```diff
@@ -33,7 +35,7 @@ To add a PagerDuty alerting destination to your pipeline:
             "payload": {
```

Comment:
Don't be afraid to just paste the payload directly from the row. Do a validation if you need to and raise an exception if message is not what is expected.

Reply:
Changed so that sink accepts a row, which is used as the payload of the PagerDuty alert.